src/MessageInterface.php

```diff
@@ -18,7 +18,11 @@ public function getId(): string;
     /**
      * Returns a job to execute
      *
-     * @return JobInterface
+     * @return string
      */
-    public function getJob(): JobInterface;
+    public function getPayloadName(): string;
```

Comment:
The message can operate PayloadInterface? getPayload(): PayloadInterface

Reply:
No, this can't be done because messages are generated not only from payload, but from unserialized queue server data too (see `Queue::run()`, `Queue::handle()`, `Queue::jobRetry()` in this PR).